tests/tests_api/test_enrollment_test.py

```diff
@@ -0,0 +1,59 @@
+from model_bakery import baker
+from rest_framework import status
+
+from api.models import EnrollmentTest, EnrollmentTestQuestionOption, Student
+
+
+def test_get_enrollment_test(api_client):
+    """Test passing enrollment test."""
+    test = baker.make(EnrollmentTest, make_m2m=True)
+    response = api_client.get(f"/api/enrollment_test/?language={test.language.id}")
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == [
+        {
+            "id": test.id,
+            "language": test.language.id,
+            "age_ranges": [age_range.id for age_range in test.age_ranges.all()],
+            "levels": [level.id for level in test.levels.all()],
```

Comment:
It's OK to return levels for the API to be universal, I think. It's just that, as I explained in the PM, for now the user won't be selecting the level themselves if a test exists.

It's up to you to remove the "levels" if you think that it will be confusing to return something that is not used at all in this iteration.

Reply:
Removed it if we really don't need it :)